addons/cargo/XEH_postInit.sqf

```diff
@@ -54,6 +54,21 @@
     [_item, "blockDamage", "ACE_cargo", false] call EFUNC(common,statusEffect_set);
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(ownerParadrop), {
+    params ["_item","_dropVelocity"];
+    
+    _item setVelocity _dropVelocity;
+}] call CBA_fnc_addEventHandler;
```

Comment:
This is just `QEGVAR(common,setVelocity)`.

Reply:
done